doc/notebook/rnn.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
RNN training requires cudnn5 and the prediction requires the pre-trained model parameters, which is not a small file and may not be suitable to put it into github.
I suggest we remove rnn and vgg. There are examples about rnn and vgg in examples/ folder (no notebooks).
I think, we can add one notebook to show the usage of tensor, initializer, optimizer, devices, and other components before regression/rbm/mlp.